gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/plugin/crud_service/ResourcePluginCrudService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.core.plugin.crud_service;
+
+import io.gravitee.apim.core.plugin.model.ResourcePlugin;
+
+public interface ResourcePluginCrudService {
+    ResourcePlugin findById(String resourceId);
```

Comment:
for me, the `findById(id)` should return an optional and `get(id)` should return the object, but it seems the initial thought has not been respected about that (and no strict rule)

Reply:
Yes, 🚀